scripts/flytekit_sagemaker_runner.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import
+import argparse
+import logging
+import subprocess
+
+parser = argparse.ArgumentParser(description="Running sagemaker task")
+parser.add_argument('--__FLYTE_SAGEMAKER_CMD__', dest='flyte_sagmaker_cmd',
```

Comment:
Are you sure this will work. Are there limits on the size of the argument?

Reply:
I don't see their CRD posing any restrictions on that front: https://github.com/aws/amazon-sagemaker-operator-for-k8s/blob/master/release/rolebased/installer.yaml#L1658

The operating system could pose a limit on the length `ARG_MAX` on a command line when the command is evaluated by the `exec` function, but that's usually in tens to hundreds of thousands of byte/characters 
https://stackoverflow.com/a/19355351
https://pubs.opengroup.org/onlinepubs/009695399/basedefs/limits.h.html

but I don't know if there's any implicit/intrinsic limitation posed by SageMaker backend

Reply:
@kumare3 
I don't see Sagemaker's CRD posing any restrictions on that front: https://github.com/aws/amazon-sagemaker-operator-for-k8s/blob/master/release/rolebased/installer.yaml#L1658

The operating system could pose a limit on the length `ARG_MAX` on a command line when the command is evaluated by the `exec` function, but that's usually in tens to hundreds of thousands of byte/characters 
https://stackoverflow.com/a/19355351
https://pubs.opengroup.org/onlinepubs/009695399/basedefs/limits.h.html

but I don't know if there's any implicit/intrinsic limitation posed by SageMaker backend

Reply:
Unfortunately, there is 
```yaml
status:
additional: "Unable to create training job: Unable to create Training Job: ValidationException:
1 validation error detected: Value '{__FLYTE_SAGEMAKER_CMD__=service_venv+pyflyte-execute+--task-module+workflows.custom_sagemaker_training+--task-name+custom_training_task+--inputs+s3://<prefix>/custom/data/inputs.pb+--output-prefix+s3://<prefix>+--my_input+hello
world}' at 'hyperParameters' failed to satisfy constraint: Map value must satisfy
constraint: [Member must have length less than or equal to 256, Member must have
length greater than or equal to 0, Member must satisfy regular expression pattern:
.*]\n\tstatus code: 400, request id: ff068295-a7ce-4fcf-8453-053a3fa9bc31"
```